tools/evaluate-sroie.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+from pathlib import Path
+from subprocess import run
+import datasets
+import time
+import tempfile
+from tqdm import tqdm
+
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.metrics import f1_score, precision_score, recall_score
+
+try:
+    import pytesseract
+except ImportError:
+    pytesseract = None
+
+
+def build_ocrs() -> None:
+    run("cargo build --release -p ocrs-cli", shell=True, check=True, text=True)
+
+
+def extract_text(image_path: str) -> str:
```

Comment:
This is fine for now. In future I think we can introduce some optimizations for running ocrs on many images in succession.